source/Sashimi.Terraform.Tests/TerraformTests.cs

```diff
@@ -0,0 +1,474 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Calamari;
+using Calamari.CloudAccounts;
+using Calamari.Integration.FileSystem;
+using Calamari.Integration.Retry;
+using Calamari.Terraform;
+using Calamari.Tests.Shared;
+using FluentAssertions;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+using Sashimi.Server.Contracts;
+using Sashimi.Server.Contracts.ActionHandlers;
+using Sashimi.Terraform.ActionHandler;
+using Sashimi.Tests.Shared;
+using Sashimi.Tests.Shared.Server;
+
+namespace Sashimi.Terraform.Tests
+{
+    [TestFixture]
+    public class TerraformTests : BaseTest
```

Comment:
The naming convention is to use `Fixture`?

Reply:
Fixed